.github/workflows/changelog.yaml

```diff
@@ -151,6 +151,9 @@ jobs:
           git add "$FILENAME"
           git commit -m "Update changelog [ci skip]"
           git push origin "$BRANCH"
+          # update the tag to include the changelog
+          git tag -fam "${GITHUB_REF##*/}" "${GITHUB_REF##*/}"
```

Comment:
A couple thoughts after doing some manual testing and looking at the [GitHub environment var doc](https://docs.github.com/en/actions/learn-github-actions/environment-variables):
* I'm not sure, but I think that this will tag every changelog change from a PR merge the PR # as well. Is that the case? If so, we might need to check `GITHUB_REF_TYPE` for `tag` first, like the "Check branch for new commits" step does
* Looks like we can use `GITHUB_REF_NAME` instead of `GITHUB_REF_##*/`

Reply:
Yup! You're right! This was supposed to be only during full releases (like the section below).